avocado/utils/process.py

```diff
@@ -158,29 +158,6 @@ def kill_process_by_pattern(pattern):
         logging.info("Succeed to run '%s'.", cmd)
 
 
-def process_in_ptree_is_defunct(ppid):
-    """
-    Verify if any processes deriving from PPID are in the defunct state.
-
-    Attempt to verify if parent process and any children from PPID is defunct
-    (zombie) or not.
-
-    :param ppid: The parent PID of the process to verify.
-    """
-    defunct = False
-    try:
-        pids = get_children_pids(ppid)
-    except CmdError:  # Process doesn't exist
-        return True
```

Comment:
I'm wondering why this returns `True` when the process does not exists. When the process does not exists it's probably not defunct, right? (although we can't inspect the children, but still it's a bit suspicious...)